src/goto-programs/remove_exceptions.cpp

```diff
@@ -323,9 +360,9 @@ void remove_exceptionst::instrument_throw(
   }
 
   // replace "throw x;" by "f#exception_value=x;"
-  exprt exc_expr=instr_it->code;
-  while(exc_expr.id()!=ID_symbol && exc_expr.has_operands())
-    exc_expr=exc_expr.op0();
+  // exprt exc_expr=instr_it->code;
+  // while(exc_expr.id()!=ID_symbol && exc_expr.has_operands())
```

Comment:
Suggest either deleting this, or adding a comment remarking on why this is disabled and under what circumstances it should be re-enabled